src/game/logic/system/damage.h

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief Damage
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+#include "always.h"
+#include "object.h"
+#include "weaponset.h"
+
+class DamageInfoInput : public SnapShot
+{
+public:
+    DamageInfoInput() :
+        m_sourceID(OBJECT_UNK),
+        m_thingTemplate(nullptr),
+        m_playerMask(0),
+        m_damageType(DAMAGE_EXPLOSION),
+        m_damageStatusType(OBJECT_STATUS_NONE),
+        m_unk1(11),
+        m_deathType(DEATH_NORMAL),
+        m_amount(0.0f),
+        m_unk2(false),
+        m_shockWaveAmount(0.0f),
+        m_shockWaveRadius(0.0f),
+        m_shockWaveTaperOff(0.0f)
+    {
+        m_shockWavePos.Zero();
+    }
+
+    virtual void CRC_Snapshot(Xfer *xfer) override {}
+    virtual void Xfer_Snapshot(Xfer *xfer) override;
+    virtual void Load_Post_Process() override {}
+
+    ObjectID m_sourceID;
+    ThingTemplate *m_thingTemplate;
+    uint16_t m_playerMask;
+    DamageType m_damageType;
+    ObjectStatusTypes m_damageStatusType;
+    unsigned int m_unk1; // seems to be some kind of enum
```

Comment:
comment that it defaults to 11, and is set to 9 in PoisonedBehavior::Update

Reply:
fixed.